src/routes/InstancesPage/InstancesPage.js

```diff
@@ -376,7 +368,7 @@ function InstancesPage() {
         <PageHeaderTitle title="ACS Instances" />
       </PageHeader>
       <Main>
-        <Card>{content}</Card>
+        {content}
```

Comment:
`<Card>` is not necessary here, and adds an additional awkward looking border when using the PF6 component.